swap/htlc.go

```diff
@@ -25,8 +26,19 @@ const (
 	HtlcNP2WSH
 )
 
+type ScriptVersion uint8
+
+const (
+	// HtlcV1 refers to the original version of the HTLC script.
+	HtlcV1 ScriptVersion = iota
```

Comment:
Are we going to need to record htlc version on disk? If so, perhaps no iota.

Reply:
No, we don't save the HTLC version. Instead we'll use the protocol version as a threshold switch for htlc v1/v2 scripts.